geometry/create-and-edit-geometries/src/main/java/com/esri/samples/create_and_edit_geometries/CreateAndEditGeometriesController.java

```diff
@@ -0,0 +1,591 @@
+import java.util.stream.Collectors;
+
+public class CreateAndEditGeometriesController {
+
+
+  @FXML
+  private MapView mapView;
+  @FXML
+  private Button pointButton;
+  @FXML
+  private Button multipointButton;
+  @FXML
+  private Button polylineButton;
+  @FXML
+  private Button polygonButton;
+  @FXML
+  private Button redoButton;
+  @FXML
+  private Button undoButton;
+  @FXML
+  public Button deleteSelectedElementButton;
+  @FXML
+  private Button stopAndSaveButton;
+  @FXML
+  private Button stopAndDiscardButton;
+  @FXML
+  public Button deleteAllGeometriesButton;
+  @FXML
+  private ComboBox<GeometryEditorTool> toolComboBox;
+
+  private GeometryEditor geometryEditor;
+  private VertexTool vertexTool;
+  private FreehandTool freehandTool;
+
+  private GraphicsOverlay graphicsOverlay;
+  private Graphic selectedGraphic;
+
+  private SimpleFillSymbol fillSymbol;
+  private SimpleLineSymbol lineSymbol;
+  private SimpleMarkerSymbol pointSymbol;
+  private SimpleMarkerSymbol multipointSymbol;
+
+  public void initialize() {
+
+    // authentication with an API key or named user is required to access basemaps and other location services
+    String yourAPIKey = System.getProperty("apiKey");
+    ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+    // create a map with the standard imagery basemap style
+    ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_IMAGERY_STANDARD);
+
+    // set the map to the map view
+    mapView.setMap(map);
+
+    // set a viewpoint on the map view
+    mapView.setViewpoint(new Viewpoint(53.08230, -9.5920, 5000));
+
+    // create a graphics overlay for the graphics
+    graphicsOverlay = new GraphicsOverlay();
+
+    // add the graphics overlay to the map view
+    mapView.getGraphicsOverlays().add(graphicsOverlay);
+
+    // create a new sketch editor and add it to the map view
+    geometryEditor = new GeometryEditor();
+    mapView.setGeometryEditor(geometryEditor);
+
+    // create vertex and freehand tools for the geometry editor and add to combo box - vertex tool selected by default
+    vertexTool = new VertexTool();
+    freehandTool = new FreehandTool();
+    toolComboBox.setConverter(new ComboBoxStringConverter());
+    toolComboBox.getItems().addAll(vertexTool, freehandTool);
+    toolComboBox.getSelectionModel().select(0);
+
+    // add example graphics to the map
+    createGraphicsMarkers();
+    createInitialGraphics();
+
+    // bind button availability to properties of the geometry editor
+    bindButtonsToGeometryEditor();
+
+    // allow user to immediately select and edit graphics
+    selectGraphic();
+  }
+
+  /**
+   * Binds button status to geometry editor properties.
+   */
+  private void bindButtonsToGeometryEditor() {
+    // point and multipoint disabled when freehand tool selected
+    pointButton.disableProperty().bind(geometryEditor.toolProperty().isEqualTo(freehandTool));
+    multipointButton.disableProperty().bind(geometryEditor.toolProperty().isEqualTo(freehandTool));
```

Comment:
```suggestion
```
We can actually remove this as the confirmed behavior here is that the combobox is only enabled once a geometry type is selected. So we can disable it by default, and then enable it if polyline/polygon are selected. More on this in another comment.